src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerFileReference.cs

```diff
@@ -462,16 +462,15 @@ public bool Equals(AnalyzerReference other)
         {
             if (other != null)
             {
-                return other.Display == this.Display &&
-                       other.FullPath == this.FullPath;
+                return other.FullPath == this.FullPath;
```

Comment:
Worth a comment about why it's not included?

Reply:
I don't think so: given it's just a computed property it never really should have been here in the first place.